### PR TITLE
Add 2.6.0 Release note.

### PR DESCRIPTION
**Pull request by codelipenghui**
### Motivation

Add 2.6.0 Release note.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@codelipenghui shall we add the following item to 2.6.0 release notes as @sijie points [here](https://github.com/apache/pulsar/pull/6122)?

## Feature
### Admin
- Change the permission level of managing subscription from super-user to tenant admin https://github.com/apache/pulsar/pull/6122

or

## Feature
### 2.6.0-SNAPSHOT (under development)
- Change the permission level of managing subscription from super-user to tenant admin https://github.com/apache/pulsar/pull/6122

?

**Comment by Anonymitaet** [comment]
Need to re-organize the contents like the following? 

### Feature/PIP

#### Broker
#### Function
#### Connector
#### Client
#### ...

### Enhancement
#### Broker
#### Function
#### Connector
#### Client
#### ...

### Bug fix
#### Broker
#### Function
#### Connector
#### Client
#### ...

### ...

**Comment by codelipenghui** [comment]
@Anonymitaet Most of bug fixes and enhancements are released in 2.5.1 and 2.5.2. So I just add a paragraph for the bug fix. So in 2.6.0 we should focus on the new features, so I did a split according to the module.

**Comment by codelipenghui** [comment]
> @codelipenghui shall we add the following item to 2.6.0 release notes as @sijie points here?

Looks #6122 is released at 2.5.1.

File: site2/website/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,124 @@
...
+
+* Added ability for sources to publish messages on their own [#6941](https://github.com/apache/pulsar/pull/6941)
+* [RabbitMQ] Allow routing key per message to RabbitMQ sink connector [#5890](https://github.com/apache/pulsar/pull/5890)
+* [RabbitMQ] Add passive config options [#6679](https://github.com/apache/pulsar/pull/6679)
+* [debezium] Upgrade from v0.10.0-Final to v1.0.0-Final [#5972](https://github.com/apache/pulsar/pull/5972)
+* [debezium] Support avro schema for debezium connector [#6034](https://github.com/apache/pulsar/pull/6034)
+* [influxdb2]  Add support for influxdb2 in pulsar-influxdb-sink [#6601](https://github.com/apache/pulsar/pull/6601)
+* [jdbc] Add jdbc sinks: postgres, mariadb, clickhouse [#6835](https://github.com/apache/pulsar/pull/6835)
+
+#### Pulsar Proxy
```
Comment (jiazhai) [inline site2/website/release-notes.md:71]: here?

File: site2/website/release-notes.md [hunk 2]
```diff
@@ -1,6 +1,124 @@
...
+* [Admin] Support delete subscription forcefully [#6383](https://github.com/apache/pulsar/pull/6383)
+* [Admin] Add subscribe initial position for consumer CLI [#6442](https://github.com/apache/pulsar/pull/6442)
+* [Admin] Support to get managed ledger info of a partitioned topic [#6532](https://github.com/apache/pulsar/pull/6532)
+* [Admin] Support compact all partitions of a partitioned topic [#6537](https://github.com/apache/pulsar/pull/6537)
+* [Admin] Support multi-hosts in PulsarAdmin [#6547](https://github.com/apache/pulsar/pull/6547)
+* [Admin] Support to get internal stats for a partitioned topic [#6624](https://github.com/apache/pulsar/pull/6624)
+* [Admin] Support enable or disable subscription auto-creation at namespace level [#6637](https://github.com/apache/pulsar/pull/6637)
+* [Admin] Enable to set the subscription expiration time for each namespace [#6851](https://github.com/apache/pulsar/pull/6851)
+
+##### Fixes
```
Comment (jiazhai) [inline site2/website/release-notes.md:107]: ####

**Comment by Anonymitaet** [comment]
As discussed w/ @codelipenghui, he will add `Change the permission level of managing subscription from super-user to tenant admin #6122` to 2.5.1 release notes in this PR.

File: site2/website/release-notes.md [hunk 3]
```diff
@@ -104,7 +224,7 @@
 * [Broker] Fixed restore clusterDispatchRate policy for compatibility [#6176](https://github.com/apache/pulsar/pull/6176) 
 * [Broker] Fix some async method problems at PersistentTopicsBase. [#6483](https://github.com/apache/pulsar/pull/6483) 
 * [Broker] This "earlier" message should be avoided to emit when reset cursor.[#6393](https://github.com/apache/pulsar/pull/6393)
-
+* [Broker] Change the permission level of managing subscription from super-user to tenant admin [#6122](https://github.com/apache/pulsar/pull/6122)
```
Comment (Anonymitaet) [inline site2/website/release-notes.md:227]: @codelipenghui 
move this to 2.5.1 release notes?

**Pull request merged**